# The Sisters Of Mercy - Covers

I analysed 996 covers of the Sisters from 820 unique artists across Spotify, Bandcamp, Soundcloud and YouTube. 

The TL;DR:

* Most covered songs: **Lucretia** by a length, then **Alice** and **Marian**.
* Most covered band line-up: **Floodland**, then **FALAA** and **Reptile House** jostling for second place.
* Busiest years for covers: **2021**, **2019** and **2020** 
* Genre: **52%** are covers in the style of the original material, then **Electronic** 15%, **Metal** 10%, **Acoustic** 9%
* Most likely to be covered by dodgy darklings: **Alice**, **Body Electric**, **Dominion**
* Most likely to be covered by techno bois: **Dominion**, **Body Electric**, **Lucretia**
* Most likely to be covered by angry metalheads: **No Time To Cry**, **This Corrosion**, **Black Planet**
* Most likely to be an acoustic/unplugged cover: **Some Kind Of Stranger**, **When You Can't See Me**, **Something Fast**
* Most diverse genres of covers: **This Corrosion**, **Temple Of Love** 
* Least diverse genres of covers: **Something Fast**, **Alice**, **Body Electric**
* Most prolific cover artists: **Brad Salyn**, followed by **nmacog** and **botchandango**
* No covers: **Watch** and **Phantom**, also **Driver**, **Wide Receiver**, **Better Reptile**, **I Will Call You**

In [ ]:
pip install requests pandas matplotlib

In [ ]:
#from google.colab import drive
#drive.mount('/content/drive')
#!cp drive/'My Drive'/Music/tsom/tsom_covers.csv .
#!ls -la

In [52]:
import requests

download_url = "https://raw.githubusercontent.com/alan-mcl/tsom-covers/main/tsom_covers.csv"
target_csv_path = "tsom_covers.csv"

response = requests.get(download_url)
response.raise_for_status()    # Check that the request was successful
with open(target_csv_path, "wb") as f:
    f.write(response.content)
print("Download ready.")
!ls -la
#!cat tsom_covers.csv

Download ready.
total 76
drwxr-xr-x 1 root root  4096 Feb 24 14:17 .
drwxr-xr-x 1 root root  4096 Feb 24 14:08 ..
drwxr-xr-x 4 root root  4096 Feb 18 14:33 .config
drwxr-xr-x 1 root root  4096 Feb 18 14:33 sample_data
-rw-r--r-- 1 root root 59548 Feb 24 14:31 tsom_covers.csv


In [53]:
import pandas as pd
temp = pd.read_csv("tsom_covers.csv", error_bad_lines=True)
tsom = temp.copy()
print(type(tsom))
print(len(tsom))
print(tsom.shape)
pd.set_option("display.max_rows", None)
tsom.head()

<class 'pandas.core.frame.DataFrame'>
987
(987, 9)


/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py:2882: FutureWarning: The error_bad_lines argument has been deprecated and will be removed in a future version.


  exec(code_obj, self.user_global_ns, self.user_ns)


Artist Title  ArtistGenre  ... bandcamp  soundcloud  youtube
0            Dreadful Shadows  1959  Unspecified  ...        0           0        1
1                         Mac  1959  Unspecified  ...        0           0        1
2                    Mad Shad  1959  Unspecified  ...        0           0        1
3                    madsmith  1959  Unspecified  ...        0           0        1
4  Maryann Giselle Alienfairy  1959  Unspecified  ...        0           0        1

[5 rows x 9 columns]

In [54]:
tsom.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 987 entries, 0 to 986
Data columns (total 9 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   Artist       987 non-null    object
 1   Title        987 non-null    object
 2   ArtistGenre  987 non-null    object
 3   CoverType    987 non-null    object
 4   Year         987 non-null    int64 
 5   spotify      987 non-null    int64 
 6   bandcamp     987 non-null    int64 
 7   soundcloud   987 non-null    int64 
 8   youtube      987 non-null    int64 
dtypes: int64(5), object(4)
memory usage: 69.5+ KB


In [55]:
tsom.describe(include=object)

Artist                    Title  ArtistGenre CoverType
count          987                      987          987       987
unique         818                       70           39        17
top     Brad Salyn  Lucretia, My Reflection  Unspecified  Straight
freq            20                      154          508       518

In [56]:
all_titles = ["1959", "A Rock And A Hard Place", "Adrenochrome", "Afterhours", "Alice", "Amphetamine Logic", "Anaconda", "Arms", "Better Reptile", "Black Planet", "Black Sail", "Blood Money", "Body And Soul", "Body Electric", "Burn", "Bury Me Deep", "But Genevieve", "Colours", "Come Together", "Crash And Burn", "Detonation Boulevard", "Doctor Jeep", "Dominion", "Driven Like The Snow", "Driver", "Finland Red, Egypt White", "First And Last And Always", "Fix", "Flood I", "Flood II", "Floorshow", "Giving Ground", "Good Things", "Heartland", "I Have Slept With All The Girls In Berlin", "I Was Wrong", "I Will Call You", "Jihad", "Kiss The Carpet", "Lights", "Lucretia, My Reflection", "Marian", "Mother Russia", "More", "Never Land", "Nine While Nine", "No Time To Cry", "On The Wire", "Phantom", "Poison Door", "Possession", "Rain From Heaven", "Ribbons", "Romeo Down", "Show Me", "Some Kind Of Stranger", "Something Fast", "Still", "Summer", "Temple Of Love", "The Damage Done", "This Corrosion", "Torch", "Train", "Under The Gun", "Valentine", "Vision Thing", "Walk Away", "War On Drugs", "Watch", "We Are The Same, Susanne", "When You Don't See Me", "Wide Receiver", "Will I Dream?", "You Could Be The One"]
lineup_map = { \
              "1959" : "Floodland",
"A Rock And A Hard Place" : "FALAA",
"Adrenochrome" : "Reptile House",
"Afterhours" : "FALAA",
"Alice" : "Reptile House",
"Amphetamine Logic" : "FALAA",
"Anaconda" : "Reptile House",
"Arms" : "Post-VT Live Only",
"Better Reptile" : "Post-VT Live Only",
"Black Planet" : "FALAA",
"Black Sail" : "Post-VT Live Only",
"Blood Money" : "FALAA",
"Body And Soul" : "FALAA",
"Body Electric" : "Reptile House",
"Burn" : "Reptile House",
"Bury Me Deep" : "FALAA",
"But Genevieve" : "Post-VT Live Only",
"Colours" : "Floodland",
"Come Together" : "Post-VT Live Only",
"Crash And Burn" : "Post-VT Live Only",
"Detonation Boulevard" : "Vision Thing",
"Doctor Jeep" : "Vision Thing",
"Dominion" : "Floodland",
"Driven Like The Snow" : "Floodland",
"Driver" : "Reptile House",
"Finland Red, Egypt White" : "Floodland",
"First And Last And Always" : "FALAA",
"Fix" : "Reptile House",
"Flood I" : "Floodland",
"Flood II" : "Floodland",
"Floorshow" : "Reptile House",
"Giving Ground" : "Floodland",
"Good Things" : "Reptile House",
"Heartland" : "Reptile House",
"I Have Slept With All The Girls In Berlin" : "Post-VT Live Only",
"I Was Wrong" : "Vision Thing",
"I Will Call You" : "Post-VT Live Only",
"Jihad" : "Floodland",
"Kiss The Carpet" : "Reptile House",
"Lights" : "Reptile House",
"Lucretia, My Reflection" : "Floodland",
"Marian" : "FALAA",
"Mother Russia" : "Floodland",
"More" : "Vision Thing",
"Never Land" : "Floodland",
"Nine While Nine" : "FALAA",
"No Time To Cry" : "FALAA",
"On The Wire" : "FALAA",
"Phantom" : "Reptile House",
"Poison Door" : "FALAA",
"Possession" : "FALAA",
"Rain From Heaven" : "Floodland",
"Ribbons" : "Vision Thing",
"Romeo Down" : "Post-VT Live Only",
"Show Me" : "Post-VT Live Only",
"Some Kind Of Stranger" : "FALAA",
"Something Fast" : "Vision Thing",
"Still" : "Post-VT Live Only",
"Summer" : "Post-VT Live Only",
"Temple Of Love" : "Reptile House",
"The Damage Done" : "Damage/Watch",
"This Corrosion" : "Floodland",
"Torch" : "Floodland",
"Train" : "FALAA",
"Under The Gun" : "Post-VT Live Only",
"Valentine" : "Reptile House",
"Vision Thing" : "Vision Thing",
"Walk Away" : "FALAA",
"War On Drugs" : "Post-VT Live Only",
"Watch" : "Damage/Watch",
"We Are The Same, Susanne" : "Post-VT Live Only",
"When You Don't See Me" : "Vision Thing",
"Wide Receiver" : "FALAA",
"Will I Dream?" : "Post-VT Live Only",
"You Could Be The One" : "Vision Thing",              
}

In [57]:
tsom["Title"] = pd.Categorical(tsom["Title"], categories=all_titles)
tsom["ArtistGenre"] = pd.Categorical(tsom["ArtistGenre"])
tsom["CoverType"] = pd.Categorical(tsom["CoverType"])

In [58]:
tsom["lineup"] = tsom["Title"].map(lineup_map)

# Methodology

Basically, I did a lot of detailed searching on each platform. I worked through the list of released and unreleased TSOM and Sisterhood tracks making queries for covers of each. I tried a variety of alternate names for the track and band where applicable. 

Where the platforms supported public playlists, and there are Sisters playlists, I perused all of those for applicable covers. 

Finally, I dug through old Heartland threads about covers and picked up some material that I missed, although often old links were dead.

"Lucretia, My Reflection" has the most variety of annoying incorrect spellings, although I didn't record that data. Bandcamp was most irritating platform because it doesn't support playlists.

Despite all this I'm sure I missed some material and expect to be loudly informed about it.

On all platforms I **disqualified** the following material as a "cover" and excluded it from the data set:

* Live or studio bootlegs of the real band.
* Rips of released material.
* Techno remixes of original material.
* Vocal covers over original material (please don't ever do this on YouTube).
* Mashups of original material (sorry Project Kiss Kass).
* "Covers of the Sisters" *own* covers (e.g. Emma; more common than you'd think).
* Solo instrumental parts (sorry Wayne and all the YouTube bassists).
* Speed Kings and NME.
* Music from those *goddam* karaoke vendors.
* Covers of that Leonard Cohen song
* Abrimaal

# Songs

Let's look at total covers.

Everyone and his dog has covered **Lucretia** (154), and this *after* I excluded countless "watch me play the Lucretia bassline" YouTube videos. **Alice** (95) and **Marian** (71) are clear second and third places, then **Temple Of Love** (59) and **This Corrosion** (58) are neck and neck. **No Time To Cry** (28) may be a surprise at number 6. **First And Last And Always** (27), **Dominion** (27), **Black Planet** (24) and **Something Fast** (24) round out the top 10.

**Driven Like The Snow** (17) and **1959** (15) make strong showings for songs that have never been performed in the live set, speaking to the enduring strength of the Floodland material.

For one of the better songs on its album **Possession** (3) has surprisingly few covers. Is this one hard to play, or hard to sing? Same could be said about **Ribbons** (6).

The Steinman collaborations (This Corrosion, Dominion/Mother Russia and More) account for 10.6% of the total covers.

Of the released material, I didn't find any covers of **Watch** and **Phantom**. I'm not surprised by the latter. But Watch deserves a cover or two - it lasted longer in the live set than Damage and the later iterations were better than the released single.

Including unreleased material: **Driver**, **Wide Receiver**, **Better Reptile** and **I Will Call You** are not represented in the data set (I didn't consider Body Politic and Burn It Down to be unique tracks). Wide Receiver is garbage, but Driver is a good song and I was surprised not to find any covers. For the two recent songs I guess we need Von to tell us the actual lyrics before anyone tries them. Maybe a cover that embraces the indistinct mumbling is the way to go?

In [59]:
tsom["Title"].value_counts()

Lucretia, My Reflection                      154
Alice                                         95
Marian                                        71
Temple Of Love                                59
This Corrosion                                58
No Time To Cry                                28
First And Last And Always                     27
Dominion                                      27
Black Planet                                  24
Something Fast                                23
When You Don't See Me                         22
Body Electric                                 21
More                                          20
Heartland                                     18
Walk Away                                     18
Vision Thing                                  18
Some Kind Of Stranger                         18
Nine While Nine                               17
Driven Like The Snow                          17
I Was Wrong                                   14
1959                

In [60]:
tsom["Title"].value_counts(normalize=True).mul(100).round(1).astype(str) + '%'

Lucretia, My Reflection                      15.6%
Alice                                         9.6%
Marian                                        7.2%
Temple Of Love                                6.0%
This Corrosion                                5.9%
No Time To Cry                                2.8%
First And Last And Always                     2.7%
Dominion                                      2.7%
Black Planet                                  2.4%
Something Fast                                2.3%
When You Don't See Me                         2.2%
Body Electric                                 2.1%
More                                          2.0%
Heartland                                     1.8%
Walk Away                                     1.8%
Vision Thing                                  1.8%
Some Kind Of Stranger                         1.8%
Nine While Nine                               1.7%
Driven Like The Snow                          1.7%
I Was Wrong                    

# Genres
I assigned a genre to each cover after listening to it. This part of the analysis is obviously somewhat subjective so take it with a pinch of salt.

"Straight" covers are those done in the style of the original material, otherwise I picked from broad genre buckets. 

53% are **Straight** covers. These are all the cover bands and tribute acts, many of the fan-made amateur covers, and a variety of professional dodgy darklings. And techno fans of The Sisterhood.

Nearly 15% are from some "**Electronic**" genre (when the original track isn't). No I don't care about the difference between your acid house vs synth pop. "Chiptune"-style covers are in this bucket too.

"**Metal**" is next at 12%. I did initially break out a couple of subgenres mostly because I am more familiar with the landscape here. Eventually I just rolled them up into the Metal bucket.

"**Acoustic**" is fourth at 9%. I included unplugged guitar, piano, and random unpowered instruments like ukulele, violin etc in here.

"**Alternative**" (6%) is the last significant category and is basically defined by the "alternative" recommendations that Bandcamp sends me weekly. A lot of artists self-identify as some kind of "alternative" too so this isn't completely subjective.

Industrial, A Capella, Punk, Country, Hip Hop, LoFi, Reggae, Soul and Jazz represent the long tail of genres with very few covers each.

In [61]:
tsom["CoverType"].value_counts(normalize=True).mul(100).round(1).astype(str) + '%'

Straight       52.5%
Electronic     14.8%
Metal          11.6%
Acoustic        9.3%
Alternative     6.2%
Darkwave        2.6%
Industrial      1.0%
Punk            0.6%
Country         0.5%
Hip Hop         0.2%
Soul            0.1%
Unknown         0.1%
Jazz            0.1%
Reggae          0.1%
LoFi            0.1%
Grindcore       0.1%
Vocal           0.1%
Name: CoverType, dtype: object

# Genre by Song

Let's dig deeper into the tracks with 20 or more covers.

**Something Fast** has the most Straight covers (70%). Basically everyone in a lockdown with a guitar and a laptop drum machine put a cover of this on YouTube.

Discounting that, **Alice** (64%), **Body Electric** (62%) and **Dominion** (52%) are most likely to be in the setlist of a tribute band near you. I am surprised **Marian** (51%) is as low as fourth. **No Time To Cry** has the least Straight covers at 21%.

**Dominion** (26%), **Body Electric** (24%) and **Lucretia** (23%) are beloved of the Electronic music community. **Something Fast** has zero techno covers even though it's name has some amusing synergy with the genre. 

Fully 42% of **No Time To Cry** covers are Metal. There is a cluster of metalheads who have redone the well-known Cradle Of Filth cover (not all of who seem to realise it's originally from the Sisters) making it arguably the most influential Sisters cover in any genre. **This Corrosion** (28%), **Black Planet** (25%) and **More** (25%) are other favourites in this genre. **Alice** (2%) has the fewest metal covers.

People with a guitar and an emo reach for **Some Kind Of Stranger** - 35% of its covers are Acoustic. **When You Don't See Me** (23%), **Something Fast** (21%) and **Marian** (18%) are obvious acoustic covers, **This Corrosion** (17%)maybe less so at #5. **Body Electric** has no acoustic covers and that is something I'd like to hear.

In [62]:
filtered = tsom.groupby("Title").filter(lambda x : len(x) >= 20)

print("Straight covers")
print("-------------------")
print((filtered[filtered["CoverType"]=="Straight"]["Title"].value_counts() / filtered["Title"].value_counts() *100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nElectronic covers")
print("-------------------")
print((filtered[filtered["CoverType"]=="Electronic"]["Title"].value_counts() / filtered["Title"].value_counts() *100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nMetal covers")
print("-------------------")
print ((filtered[filtered["CoverType"]=="Metal"]["Title"].value_counts() / filtered["Title"].value_counts() *100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nAcoustic covers")
print("-------------------")
print((filtered[filtered["CoverType"]=="Acoustic"]["Title"].value_counts() / filtered["Title"].value_counts() *100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')


Straight covers
-------------------
Something Fast               73.9%
Alice                        64.2%
Body Electric                61.9%
Dominion                     51.9%
Marian                       50.7%
Lucretia, My Reflection      50.6%
More                         50.0%
First And Last And Always    44.4%
When You Don't See Me        40.9%
Temple Of Love               39.0%
Black Planet                 37.5%
This Corrosion               24.1%
No Time To Cry               21.4%
1959                          nan%
A Rock And A Hard Place       nan%
Adrenochrome                  nan%
Afterhours                    nan%
Amphetamine Logic             nan%
Anaconda                      nan%
Arms                          nan%
Name: Title, dtype: object

Electronic covers
-------------------
Dominion                     25.9%
Body Electric                23.8%
Lucretia, My Reflection      22.7%
No Time To Cry               21.4%
Black Planet                 20.8%
Alice                  

# Diversity

I hacked up a "diversity index" for each song by using the sum of squares of the proportions of each genre per song, inverted because big numbers are better.

Once again taking the songs with more than 20 total covers, **This Corrosion** stands out with the most diverse group of covers. Something about it's combination of cocky beats, relentless groove and over-the-top angst clearly appeals to a wide range of artists.

**Temple Of Love** is in a clear second place, also attracting the attention of a wide variety of musical styles. **Black Planet**, **First And Last And Always** and **When You Don't See Me** round out the "diversity" top 5.

**Something Fast** props up the bottom of the list because all it's covers are either straight or acoustic (i.e. straight without drums). **Alice** and **Body Electric** are a predictably second and third least diverse.

In [63]:
filtered = tsom.groupby("Title").filter(lambda x : len(x) >= 20)

(filtered.groupby(["Title", "CoverType"])["CoverType"].count()/tsom.groupby(["Title"])["Title"].count()).pow(2).groupby("Title").sum().pow(0.5).pow(-1).sort_values(ascending=False)


Title
1959                                              inf
Show Me                                           inf
I Will Call You                                   inf
A Rock And A Hard Place                           inf
Kiss The Carpet                                   inf
Lights                                            inf
Mother Russia                                     inf
Never Land                                        inf
Nine While Nine                                   inf
On The Wire                                       inf
Phantom                                           inf
Poison Door                                       inf
Possession                                        inf
Rain From Heaven                                  inf
Ribbons                                           inf
Romeo Down                                        inf
Some Kind Of Stranger                             inf
I Have Slept With All The Girls In Berlin         inf
Still                 

# Line-up
Sorting total covers by era, the **Floodland** "line-up" (314) is well out ahead as the most covered, mostly due to the volume of Lucretia (note that I included The Sisterhood tracks in this line-up bucket which may not be entirely fair). **FALAA** (266) and **Reptile House** (263)are pretty even behind that. Despite being an actual live line-up the **Vision Thing** songs (112) are a lot less popular with covering artists. One could make a case for splitting the post-VT live line-ups by lead guitarist or something but I couldn't be arsed.

All line-ups have roughly the same proportion of straight covers. Floodland unsurprisingly has the highest proportion of Electronic covers (18%). Vision Thing has both the most Acoustic (16%) and Metal (13%) covers, which might speak to its reach outside the traditional Sisters fanbase.

In [64]:
tsom["lineup"].value_counts()

Floodland            312
FALAA                263
Reptile House        261
Vision Thing         110
Post-VT Live Only     36
Damage/Watch           4
Name: lineup, dtype: int64

In [65]:
filtered = tsom

print("\nFloodland")
print("-------------------")
print((filtered[filtered["lineup"]=="Floodland"].groupby("CoverType")["Title"].count()/filtered[filtered["lineup"]=="Floodland"]["Title"].count()*100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nFALAA")
print("-------------------")
print((filtered[filtered["lineup"]=="FALAA"].groupby("CoverType")["Title"].count()/filtered[filtered["lineup"]=="FALAA"]["Title"].count()*100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nReptile House")
print("-------------------")
print((filtered[filtered["lineup"]=="Reptile House"].groupby("CoverType")["Title"].count()/filtered[filtered["lineup"]=="Reptile House"]["Title"].count()*100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')

print("\nVision Thing")
print("-------------------")
print((filtered[filtered["lineup"]=="Vision Thing"].groupby("CoverType")["Title"].count()/filtered[filtered["lineup"]=="Vision Thing"]["Title"].count()*100).sort_values(ascending=False).head(20).round(1).astype(str) + '%')



Floodland
-------------------
CoverType
Straight       47.4%
Electronic     18.3%
Metal          11.9%
Acoustic        8.0%
Alternative     7.7%
Darkwave        2.2%
Country         1.6%
Hip Hop         0.6%
Industrial      0.6%
Soul            0.3%
Unknown         0.3%
Jazz            0.3%
Punk            0.3%
Vocal           0.3%
Reggae          0.0%
LoFi            0.0%
Grindcore       0.0%
Name: Title, dtype: object

FALAA
-------------------
CoverType
Straight       51.0%
Electronic     14.8%
Metal          14.8%
Acoustic       10.6%
Alternative     5.7%
Darkwave        1.9%
Industrial      0.8%
Grindcore       0.4%
Reggae          0.0%
Unknown         0.0%
Soul            0.0%
Jazz            0.0%
Punk            0.0%
LoFi            0.0%
Hip Hop         0.0%
Country         0.0%
Vocal           0.0%
Name: Title, dtype: object

Reptile House
-------------------
CoverType
Straight       55.9%
Electronic     14.9%
Metal           8.8%
Acoustic        7.7%
Alternative     5.4%
Dark

In [66]:
tsom["Year"].value_counts(normalize=True).mul(100).round(1).astype(str) + '%'

2021    14.2%
2019    10.2%
2020     9.3%
2015     8.1%
2017     7.8%
2016     7.6%
2013     7.0%
2012     6.7%
2018     6.6%
2014     6.3%
2011     3.7%
2010     2.3%
2022     1.9%
2000     1.8%
2008     1.5%
2007     1.4%
2009     1.2%
2001     0.4%
2006     0.2%
1993     0.2%
1995     0.2%
1996     0.2%
1997     0.2%
2005     0.2%
1998     0.1%
2003     0.1%
1992     0.1%
2004     0.1%
1991     0.1%
2002     0.1%
Name: Year, dtype: object

# Artist

The median Sisters covers per artist is 1. Most people are happy doing one Sisters cover and then moving on with their lives. The mean is 1.2 but that is propped up by some outliers at the top.

81 artists covered 2 or more songs. **Brad Salyn** (20) is the most prolific. **nmacog** (15) and **botchandango** (10) are the others in double digits. Look these guys up on Soundcloud and YouTube.

This data doesn't represent the tribute bands very well because of the limited vids of them that exist on YouTube.



In [67]:
tsom["Artist"].value_counts().describe()

count    818.000000
mean       1.206601
std        1.065817
min        1.000000
25%        1.000000
50%        1.000000
75%        1.000000
max       20.000000
Name: Artist, dtype: float64

In [68]:
print(tsom["Artist"].value_counts()[lambda x : x>=2].count())
tsom["Artist"].value_counts().head(10)


80


Brad Salyn                   20
Nmacog                       15
Botchandango                 10
madsmith                      9
Zombie Thirteen               7
Blumenhofen                   6
Temple of Mercy               5
Tobias Forsner/leftydrake     4
The Sisters Of Murphy         4
Paranoid Android              4
Name: Artist, dtype: int64

# Year

**2021** was the biggest year by far for Sisters covers (141), followed by **2019** (101) and **2020** (92). In general, the number of Sisters covers each year has grown almost every year that I have data for, with a noticeable spike in 2000 when that unspeakable tribute album came out. 

Some of this is recency bias: more recent covers are easier to find and more likely to still be available on the platforms. But the pandemic effects of the last few years and the increasing ease of making, recording and publishing music online certainly have a role too.



In [69]:
tsom.groupby("Year")["Year"].count()

Year
1991      1
1992      1
1993      2
1995      2
1996      2
1997      2
1998      1
2000     18
2001      4
2002      1
2003      1
2004      1
2005      2
2006      2
2007     14
2008     15
2009     12
2010     23
2011     37
2012     66
2013     69
2014     62
2015     80
2016     75
2017     77
2018     65
2019    101
2020     92
2021    140
2022     19
Name: Year, dtype: int64

# Platform

**Spotify** has the least covers (72). These are mostly established professional artists  with record company deals. **Bandcamp** (81) has only slightly more. I suppose these artists are also professional but less established and/or successful.

**SoundCloud** (401) has lots of covers. Most are solidly in the realm of amateur (disclosure: my own included), or at least "musician isn't my only job" artists. 

**YouTube** (620) has the most covers. This is basically the full gamut from pros who are also on Spotify, to less established bands, to talented amateurs, to your uncle Bob and his dog covering Something Fast.

There was less overlap that I expected. 85% of covers were on one platform only and 13% were on two platforms. I found only *3* total covers on all four platforms - shout out to the marketing teams for Cradle Of Filth, The Court Of Sybaris and Sleepmask.

Rehashing the diversity index (heh) suggests the **Bandcamp** hosts the least diversity of covers, which is disappointing. For whatever reason, the covers there are largely straight derivatives of the original material. **Spotify** and **SoundCloud** are better. I expected SoundCloud to top the diversity stakes but that honour is held by **YouTube**, even after applying my strict filtering criteria.

In [70]:
tsom.filter(["spotify", "bandcamp", "soundcloud", "youtube"]).sum()

spotify        71
bandcamp       80
soundcloud    395
youtube       619
dtype: int64

In [71]:
tsom["platforms"] = tsom["spotify"] + tsom["bandcamp"] + tsom["soundcloud"] + tsom["youtube"]

print(tsom.groupby("platforms")["Title"].count())

tsom.groupby("platforms")["Title"].count() / len(tsom) *100



platforms
1    833
2    132
3     18
4      3
Name: Title, dtype: int64


platforms
1    84.397163
2    13.373860
3     1.823708
4     0.303951
Name: Title, dtype: float64

In [72]:
filtered = tsom[tsom["spotify"] == 1]
print(((filtered.groupby(["CoverType"])["CoverType"].count()/len(filtered)).pow(2).sum()) ** 0.5 ** -1 *100)
filtered = tsom[tsom["bandcamp"] == 1]
print(((filtered.groupby(["CoverType"])["CoverType"].count()/len(filtered)).pow(2).sum()) ** 0.5 ** -1 *100)
filtered = tsom[tsom["soundcloud"] == 1]
print(((filtered.groupby(["CoverType"])["CoverType"].count()/len(filtered)).pow(2).sum()) ** 0.5 ** -1 *100)
filtered = tsom[tsom["youtube"] == 1]
print(((filtered.groupby(["CoverType"])["CoverType"].count()/len(filtered)).pow(2).sum()) ** 0.5 ** -1 *100)

7.289250168062473
7.0556640625
9.468760266762985
12.0369396673852


# Conclusion

Other than concluding that I spent way too much time on this, I will leave further interpretation up to the reader. It would be interesting to compare Sisters covers to those from other bands, but it sure ain't gonna be me who collects that data.

I will follow this up with a post listing some of my favourite covers discovered during this exercise.

PM me if you want to get hold of the data set and the Jupyter notebook (I used Google Colab and Pandas).